src/yamlfix/adapters.py

```diff
@@ -452,8 +452,7 @@ def _fix_top_level_lists(source_code: str) -> str:
 
         return "\n".join(fixed_source_lines)
 
-    @staticmethod
```

Comment:
Why did you change the signature of the method? I see that you're not using `self` inside the method so probably this can be set back to a `@staticmethod`